lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -1,27 +1,31 @@
 .wrapper-preview-menu {
   @include clearfix();
-  @include box-sizing(border-box);
+
   margin: 0 auto 0;
   padding: ($baseline*0.75);
-  background-color: $gray-l3;
+  background-color: $lms-preview-menu-color;
+  box-sizing: border-box;
 
   @media print {
     display: none;
   }
 
   .preview-menu {
     @extend %inner-wrapper;
+
     width: auto;
   }
 
   .preview-actions {
     display: inline-block;
+    margin-bottom: 0;
```

Comment:
add - @include padding-left(0); to align staff bar
![image](https://user-images.githubusercontent.com/2023680/31292989-8f4dfe24-aaa3-11e7-940e-7fa11bc970f0.png)

Reply:
Fixed